front/app/components/admin/NavigationTabs/index.tsx

```diff
@@ -11,8 +12,7 @@ const NavigationTabs = styled.nav`
   ${({ theme }) => css`
     position: fixed;
     width: 100%;
-    // TODO : set bg color in component library
-    background: #fbfbfb;
+    background-color: ${lighten('0.05', colors.adminBackground)};
```

Comment:
Not sure how often we're doing this in the code these days, I don't think that often, think we're rather trying to standardize the palette. Functions like lighten break if you try to do e.g. a dark mode, or might not work out well with certain colors. I'm sure we're doing it sometimes, and I'm okay with it, just asking the question.

Probably related and to be taken up wrt the design system we're thinking about.